interpreter/cling/lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -207,7 +207,8 @@ class Azog: public RTDyldMemoryManager {
 
   void deregisterEHFrames() override {
 #ifdef LLVM_ON_WIN32
-    platform::DeRegisterEHFrames(Addr, Size);
+    // platform::DeRegisterEHFrames(Addr, Size);
```

Comment:
@marsupial, should I bring back the arguments of `deregisterEHFrames` or we could update `platform::DeRegisterEHFrames`'s signature?